camply/providers/recreation_dot_gov/recdotgov_tours.py

```diff
@@ -23,15 +28,36 @@
 logger = logging.getLogger(__name__)
 
 
-class RecreationDotGovTours(RecreationDotGovBase):
+class RecreationDotGovTours(RecreationDotGovBase, ABC):
+    """
+    Recreation.gov Implementation for Tours
+    """
+
     resource_api_path = RIDBConfig.TOUR_API_PATH
     api_response_class = TourResponse
     api_search_result_class = RecDotGovSearchResult
     api_search_result_key = "entity_id"
+    activity_name = "TOURS"
```

Comment:
this can't be TOURS and was left as None on purpose (to stop sending `&activity=` in `https://ridb.recreation.gov:443 "GET /api/v1/facilities?full=true&state=CO&activity=CAMPING&offset=0 HTTP/1.1"`) - it appears most of tours don't have an activity specified so there's no way to filter on this.

Reply:
Oh, you're right. We can set this to `None`. 

I'm trying to wrap my brain around this though: when we set `activity_name = None` - where does it know to filter for relevant content? This just seems to magically work without passing other query parameters. Does it throw out bad results at the JSON parsing step?

```commandline
poetry run camply --debug campgrounds --provider RecreationDotGovDailyTicket --state CA

[2022-11-28 14:00:37] CAMPLY   camply, the campsite finder ⛺️
[2022-11-28 14:00:37] DEBUG    Setting up camply debugging
[2022-11-28 14:00:37] DEBUG    Camply Version: 0.7.1
[2022-11-28 14:00:37] DEBUG    Python Version: 3.10.8
[2022-11-28 14:00:37] DEBUG    Platform: darwin
[2022-11-28 14:00:37] DEBUG    Starting new HTTPS connection (1): ridb.recreation.gov:443
[2022-11-28 14:00:42] DEBUG    https://ridb.recreation.gov:443 "GET /api/v1/facilities?full=true&state=CA&offset=0 HTTP/1.1" 200 None
[2022-11-28 14:00:42] DEBUG    Starting new HTTPS connection (1): ridb.recreation.gov:443
[2022-11-28 14:00:46] DEBUG    https://ridb.recreation.gov:443 "GET /api/v1/facilities?full=true&state=CA&offset=1000 HTTP/1.1" 200 None
[2022-11-28 14:00:47] INFO     Too Many Results returned (2007), try performing a more specific search
[2022-11-28 14:00:47] INFO     10 Matching Campgrounds Found
[2022-11-28 14:00:47] INFO     ⛰  Carrizo Plain National Monument, CA (#3112) - 🏕  Carrizo Plain National Monument Tours (#233244)
[2022-11-28 14:00:47] INFO     ⛰  Death Valley National Park, CA (#2662) - 🏕  Death Valley National Park Ranger Programs (#10088452)
[2022-11-28 14:00:47] INFO     ⛰  Death Valley National Park, CA (#2662) - 🏕  Scotty'S Castle - Death Valley National Park (#233282)
[2022-11-28 14:00:47] INFO     ⛰  Fort Point National Historic Site, CA (#2698) - 🏕  Fort Point National Historic Site Tours (#10088307)
[2022-11-28 14:00:47] INFO     ⛰  Golden Gate National Recreation Area, CA (#2730) - 🏕  Golden Gate National Recreation Area Tours (#10088963)
[2022-11-28 14:00:47] INFO     ⛰  Lava Beds National Monument, CA (#2797) - 🏕  Lava Beds National Monument Tours (#251978)
[2022-11-28 14:00:47] INFO     ⛰  Piedras Blancas Historic Lightstation Outstanding Natural Area, CA (#10008690) - 🏕  Piedras Blancas Light Station (#252948)
[2022-11-28 14:00:47] INFO     ⛰  San Francisco Maritime National Historical Park, CA (#2915) - 🏕  San Francisco Maritime National Historic Park Tours (#249985)
[2022-11-28 14:00:47] INFO     ⛰  Sequoia & Kings Canyon National Parks, CA (#2931) - 🏕  Crystal Cave Tours Sequoia National Park (#251898)
[2022-11-28 14:00:47] INFO     ⛰  Whiskeytown National Recreation Area, CA (#2977) - 🏕  Whiskeytown National Recreation Area Ranger-Guided Kayak And Paddle Board Tours (#274349)
[2022-11-28 14:00:47] CAMPLY   Exiting camply 👋
```

Reply:
It doesn't filter facilities server-side (like camping). FacilityResponse should be able to hold whatever facility (there's nothing specific to camping or tours in FacilityResponse). When all facilities are fetched, they're filtered client-side by https://github.com/jingyuanliang/camply/blob/6736fa7300cf3c4f9157e3585399519b6acc8c33/camply/providers/recreation_dot_gov/recdotgov_provider.py#L407

Reply:
Will you fix this or is it on me?

Reply:
I just fixed it - thanks for all of the clarification here.